src/m365/base/ContextCommand.ts

```diff
@@ -0,0 +1,40 @@
+import * as fs from 'fs';
+import AnonymousCommand from './AnonymousCommand';
+import { Logger } from '../../cli/Logger';
+import { M365RcJson } from './M365RcJson';
+
+export default abstract class ContextCommand extends AnonymousCommand {
+  public removeContextInfo(logger: Logger): void {
+    const filePath: string = '.m365rc.json';
```

Comment:
Before removing anything, we should prompt the user for confirmation like we do in all other remove commands